Sami/Resources/themes/default/sami.js.twig

```diff
@@ -1,5 +1,9 @@
 {% from _self import element %}
 
+{% block project_version %}
```

Comment:
I was lucky, because I worked just like that without any knowledge of twig :)
After reading twig's doc further, you re probably right